src/NotifyIconWpf/TaskbarIcon.cs

```diff
@@ -762,6 +794,8 @@ private void ShowContextMenu(Point cursorPosition)
             // fallback, the context menu can't receive keyboard events - should not happen though
             WinApi.SetForegroundWindow(handle);
 
+            ContextMenu.Focus();
```

Comment:
I'd like to understand why you have added a focus, can you add some comment here?

Reply:
I still would like to know why you think this is needed, was there some kind of issue without it?

Reply:
Without this line, the context menu doesn't have keyboard focus after being displayed (at least on Windows 11), so navigation with the arrow keys doesn't work, and you can't even close the context menu with Escape. This line should definitely be merged into the main branch, even without the rest of the patch.